release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -184,6 +184,7 @@ This release introduces the following updates to the *Administrator* perspective
 
 * Enable and disable the tailing to Pod log viewer to minimize load time.
 * View recommended values for `VerticalPodAutoscaler` on the *Deployment* page.
+* Disable `BuildConfig` and `DeploymentConfigs` when a cluster does not have a `build.openshift.io/v1` API version.
```

Comment:
DeploymentConfig is a separate API, controlled by a separate cluster capability (`DeploymentConfig`). If DeploymentConfig web console components are removed simply because the `build.openshift.io` API group is not present, this is a bug.

Reply:
@adambkaplan Just to elaborate the issue, we observed that when there is no build.openshift.io/v1` API version in the cluster, the DeployConfig does not work properly. We did a small spike to see that although the ReplicationController for the DC is getting created, still DC is not starting the pods. This hinted some underlining connection b/w the BC and DC. 
But surely it may also be a bug. It would be great if we can ask someone from the respective team.

Reply:
> But surely it may also be a bug. It would be great if we can ask someone from the respective team.

This is absolutely a bug. Potentially related to (or fixed by) [OCPBUGS-22956](https://issues.redhat.com/browse/OCPBUGS-22956)

Reply:
Looks like someone wrote up text for that bug, but it didn't make it into the release notes. I have reached out to the writer working in this area for more information.

I will most likely update this PR with a note for OCPBUGS-22956 in the bug fixes section of the release notes based on what I find out.

Reply:
Updated to include the bug fix.